CowLite Audioplayer/src/cap/util/QueryReader.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package cap.util;
+
+import java.net.URLDecoder;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Objects of this class can be used to decode queries into a map in which
+ * the key is a variable name in the query and the value is a value of the given variable name.
+ * @author Wessel Jongkind
+ * @version 16-06-2017
+ */
+public class QueryReader
```

Comment:
Styling/indentation

Reply:
Also, consider moving this utility into separate .jar as it's used in multiple places along with SugarySyntax file.